bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/model/Precipitation.java

```diff
@@ -27,8 +27,7 @@
             @Provider(name = ProviderName.WUNDERGROUND, property = "current_observation.precip_1hr_metric"),
             @Provider(name = ProviderName.WUNDERGROUND, property = "qpf_allday.mm"),
             @Provider(name = ProviderName.OPENWEATHERMAP, property = "rain.3h", converter = ConverterType.DOUBLE_3H),
-            @Provider(name = ProviderName.OPENWEATHERMAP, property = "rain", converter = ConverterType.DOUBLE_3H),
-            @Provider(name = ProviderName.OPENWEATHERMAP, property = "rain.1h"),
```

Comment:
Why is this rain.1h value removed? Is it unused?

Reply:
In the API docs I can't find anything about a "rain.1h" property. My latest API calls didn't show it either in the responses. I don't know if it produces overhead if we leave it in there.

Reply:
Good enough.